Installation issue: Praview needs Verdict

### Steps to reproduce the issue

```console
$ spack install paraview@master+qt+python3
```


Installing paraview crash during CMake because it does not find the Verdict lib.
Am I the only one to have this ?
To fix this, we could add ```'-DVTK_MODULE_USE_EXTERNAL_verdict=OFF'``` in the CMake args

@chuckatkins @danlipsa  @vicentebolea, do you agree ?



### Error message

```
1 error found in build log:
     225    Call Stack (most recent call first):
     226      VTK/CMake/vtkModule.cmake:5065 (vtk_module_find_package)
     227      VTK/CMake/vtkModule.cmake:4936 (vtk_module_third_party_external)
     228      VTK/ThirdParty/verdict/CMakeLists.txt:1 (vtk_module_third_party)
     229    
     230    
  >> 231    CMake Error at VTK/CMake/vtkModule.cmake:4471 (message):
     232      Could not find the Verdict external dependency.
     233    Call Stack (most recent call first):
     234      VTK/CMake/vtkModule.cmake:5065 (vtk_module_find_package)
     235      VTK/CMake/vtkModule.cmake:4936 (vtk_module_third_party_external)
     236      VTK/ThirdParty/verdict/CMakeLists.txt:1 (vtk_module_third_party)
     237    

See build log for details:
  /tmp/billae/spack-stage/spack-stage-paraview-master-ob4aut4wpm5cue2sl6ecxc5ubtphxztd/spack-build-out.txt
```


### Information on your system

* **Spack:** 0.18.0.dev0 (ac9bd3d72d7a4ed49e5bbdd3f07a4fdf91f8d0ce)
* **Python:** 3.6.9
* **Platform:** linux-ubuntu18.04-skylake
* **Concretizer:** clingo


### Additional information

_No response_

### General information

- [X] I have run `spack debug report` and reported the version of Spack/Python/Platform
- [X] I have run `spack maintainers <name-of-the-package>` and **@mentioned** any maintainers
- [X] I have uploaded the build log and environment files
- [X] I have searched the issues of this repo and believe this is not a duplicate

--- a/var/spack/repos/builtin/packages/paraview/package.py
+++ b/var/spack/repos/builtin/packages/paraview/package.py
@@ -345,6 +345,9 @@ class Paraview(CMakePackage, CudaPackage):
             '-DBUILD_TESTING:BOOL=OFF',
             '-DOpenGL_GL_PREFERENCE:STRING=LEGACY']
 
+        if spec.satisfies('@master'):
+            cmake_args.append('-DVTK_MODULE_USE_EXTERNAL_VTK_verdict:BOOL=OFF')
+
         if spec.satisfies('@5.10:'):
             cmake_args.extend([
                 '-DVTK_MODULE_USE_EXTERNAL_ParaView_vtkcatalyst:BOOL=OFF',

--- a/var/spack/repos/builtin/packages/paraview/package.py
+++ b/var/spack/repos/builtin/packages/paraview/package.py
@@ -345,7 +345,7 @@ class Paraview(CMakePackage, CudaPackage):
             '-DBUILD_TESTING:BOOL=OFF',
             '-DOpenGL_GL_PREFERENCE:STRING=LEGACY']
 
-        if spec.satisfies('@master'):
+        if spec.satisfies('@5.11:'):
             cmake_args.append('-DVTK_MODULE_USE_EXTERNAL_VTK_verdict:BOOL=OFF')
 
         if spec.satisfies('@5.10:'):

